### PR TITLE
Fixed Belarusian language entry name

### PR DESCRIPTION
**Pull request by artsiom-voitas**
Fixed Belarusian language entry name from "by" to "be", which is correct key in i18n docs, and fixed one translation value.

### CODE REVIEW

File: interface/app/$libraryId/settings/client/general.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const LANGUAGE_OPTIONS = [
 	{ value: 'fr', label: 'Français' },
 	{ value: 'tr', label: 'Türkçe' },
 	{ value: 'nl', label: 'Nederlands' },
-	{ value: 'by', label: 'Беларуская' },
+	{ value: 'be', label: 'Беларуская' },
```
Comment (utkubakir) [inline interface/app/$libraryId/settings/client/general.tsx:31]: Need more information on this. Also is this tested on Desktop? Folder name is "by", changing a language might not work here.

image

Comment (artsiom-voitas) [reply interface/app/$libraryId/settings/client/general.tsx:31]: In day.js [docs](https://cdn.jsdelivr.net/npm/dayjs@1/locale.json), they use it like `"key":"be","name":"Belarusian"`, same goes with [date-fns](https://github.com/date-fns/date-fns/tree/main/src/locale/be)
I messed up with folder name, while commit, will fix it asap

Comment (artsiom-voitas) [reply interface/app/$libraryId/settings/client/general.tsx:31]: image
I didn't mess up, we just didn't notice, the folder name was changed with the commit. And it works properly!

Comment (artsiom-voitas) [reply interface/app/$libraryId/settings/client/general.tsx:31]: image
I didn't mess up, we just didn't notice, the folder name was changed with the commit. And it works properly!

Comment (utkubakir) [reply interface/app/$libraryId/settings/client/general.tsx:31]: cool, just to confirm you tested this desktop?

Comment (artsiom-voitas) [reply interface/app/$libraryId/settings/client/general.tsx:31]: > cool, just to confirm you tested this desktop?

Yeah, it works
And just to confirm from your end, is project.inlang/settings.json gone for good? There is no other config file, which need to be used to 100% right configuration of i18n?

**Review by utkubakir — APPROVED** [review]

**Pull request merged**